### ISSUE TITLE
Learning Important Features Through Propagating Activation Differences

### ISSUE THREAD

**Issue by traversc** [greenelab/deep-review#863]
An interesting paper International Conference on Machine Learning 2017.  

They develop a method for the purpose of finding out important features in a classification model.  They applied it to a (simulated) DNA motif dataset, using a convolutional neural network classifier.  Their approach seemed to better pick out the important DNA base pairs of the model.  (Figure 6).  

Abstract:

> The purported “black box” nature of neural networks is a barrier to adoption in applications where interpretability is essential. Here we present DeepLIFT (Deep Learning Important FeaTures), a method for decomposing the output prediction of a neural network on a specific input by backpropagating the contributions of all neurons in the network to every feature of the input. DeepLIFT compares the activation of each neuron to its ‘reference activation’ and assigns contribution scores according to the difference. By optionally giving separate consideration to positive and negative contributions, DeepLIFT can also reveal dependencies which are missed by other approaches. Scores can be computed efficiently in a single backward pass. We apply DeepLIFT to models trained on MNIST and simulated genomic data, and show significant advantages over gradient-based methods. A detailed video tutorial on the method is at http://goo.gl/qKb7pL and code is at http://goo.gl/RM8jvH.

https://arxiv.org/pdf/1704.02685.pdf

**Comment by alxndrkalinin** [comment]
Duplicate of #50. This paper [is discussed](https://greenelab.github.io/deep-review/#assigning-example-specific-importance-scores) in the `Interpretation` subsection of the `Discussion` in v1.0.

**Comment by agitter** [comment]
Closing the duplicate issue so we can keep the discussion in #50.  I will note that this has been published though.
